app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
```

Comment:
- Good job done so far, but as i suggested please setup view files for each of your controller actions 👍

Reply:
noted, worked on it.